api/sales_dashboard/templates/sales_dashboard/home.html

```diff
@@ -18,9 +18,13 @@ <h1 class="h2">Organisations</h1>
             </form>
           </div>
           <div class='float-left'>
-            <form method="POST" action="{% url 'sales_dashboard:update-organisation-subscription-information-caches' %}">
+            <form method="POST" action="{% url 'sales_dashboard:update-organisation-subscription-information-influx-caches' %}">
               {% csrf_token %}
-              <input type="submit" value="Update subscription information caches" class="btn btn-primary m-1">
+              <input type="submit" value="Update subscription information caches from influx" class="btn btn-primary m-1">
+            </form>
+            <form method="POST" action="{% url 'sales_dashboard:update-organisation-subscription-information-chargebee-caches' %}">
+              {% csrf_token %}
+              <input type="submit" value="Update subscription information caches from chargebee" class="btn btn-primary m-1">
             </form>
             <small class="p-1">Last updated at: {{ subscription_information_caches_updated_at|default:"never" }}</small>
```

Comment:
Maybe we could keep it only for Influx (and probably add a new `updated_at` field for Chargebee) and add a note somewhere specifying that it is referred only to INflux and that ChargeBee is synced automatically?

Reply:
Yep, makes sense!

Reply:
> - this could have been updated by the chargebee automated process rather than the influx bulk update.
If we have two `updated_at` fields, the one for Influx will be correct since we always update Influx data in bulk, right?
Regarding Chargebee, we could store the `chargebee_updated_at` but I don't think it is worth it to show it in the dashboard right?

Reply:
Two new fields, 'influx_updated_at' and 'chargebee_updated_at', were added to the 'organisations_organisationsubscriptioninformationcache' table. The date is obtained from 'influx_updated_at'.